UlKit/UlKit/CatalogViewController.swift

```diff
@@ -3,9 +3,280 @@
+        button.setImage(UIImage(named: imageName), for: .normal)
+        if let action = action {
+            button.addTarget(self, action: action, for: .touchUpInside)
+        }
+        return button
+    }
+
+    private func createShadowView() -> UIView {
+        let view = UIView()
+        view.translatesAutoresizingMaskIntoConstraints = false
+        view.layer.cornerRadius = 10
+        view.layer.shadowColor = UIColor.black.cgColor
+        view.layer.shadowOpacity = 0.4
+        view.layer.shadowOffset = .init(width: 0, height: 5)
+        view.layer.shadowRadius = 4
+        view.contentMode = .scaleAspectFit
+        return view
+    }
+
+    private func createImageView(nameImage: String) -> UIImageView {
+        let imageView = UIImageView()
+        imageView.image = UIImage(named: nameImage)
+        imageView.contentMode = .scaleAspectFill
+        imageView.clipsToBounds = true
+        imageView.layer.cornerRadius = 10
+        imageView.translatesAutoresizingMaskIntoConstraints = false
+        return imageView
+    }
+
+    private func configElementsButton() {
+        brendsButton.titleEdgeInsets = UIEdgeInsets(top: 0, left: -340, bottom: 0, right: -10)
+        brendsButton.imageEdgeInsets = UIEdgeInsets(top: 0, left: 245, bottom: 0, right: 10)
+
+        shoesButton.titleEdgeInsets = UIEdgeInsets(top: 0, left: -345, bottom: 0, right: -10)
+        shoesButton.imageEdgeInsets = UIEdgeInsets(top: 0, left: 245, bottom: 0, right: 10)
+
+        bagButton.titleEdgeInsets = UIEdgeInsets(top: 0, left: -345, bottom: 0, right: -10)
+        bagButton.imageEdgeInsets = UIEdgeInsets(top: 0, left: 245, bottom: 0, right: 10)
+    }
+
+    private func configSegmentControl() {
+        categoriesSegmentedControl.selectedSegmentIndex = 0
+        categoriesSegmentedControl.addTarget(self, action: #selector(reactsToSwitches), for: .valueChanged)
+    }
+
+    @objc
+    private func callsCamera() {
+        let imagePicker = UIImagePickerController()
+        imagePicker.sourceType = .camera
+        imagePicker.delegate = self
+        present(imagePicker, animated: true)
+    }
+
+    @objc
+    private func selectBarcode() {
+        let imagePicker = UIImagePickerController()
+        imagePicker.sourceType = .camera
+        imagePicker.delegate = self
+        present(imagePicker, animated: true)
+    }
+
+    @objc
+    private func reactsToSwitches() {
+        switch categoriesSegmentedControl.selectedSegmentIndex {
+        case 0:
+            shoesImageView.image = UIImage(named: Constants.shoesNameImage)
+            sneakersImageView.image = UIImage(named: Constants.sandalsNameImage)
+            shoesButton.setImage(UIImage(named: Constants.shoesWomanNamePicture), for: .normal)
+            bagButton.setImage(UIImage(named: Constants.bagNameImage), for: .normal)
+        case 1:
+            shoesImageView.image = UIImage(named: Constants.manShoesNameImage)
```

Comment:
картинки их assets мы получаем через енам, не через строки
```suggestion
            shoesImageView.image = .manShoesNameImage
```